tests/components/normal-components/resonator.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { it, expect, describe } from "bun:test"
+import { getTestFixture } from "tests/fixtures/get-test-fixture"
+
+describe("Resonator Component", () => {
+  it("should render a resonator with no ground pins", async () => {
+    const { circuit } = getTestFixture()
+    circuit.add(
+      <board width="10mm" height="10mm">
+        <resonator
+          name="R1"
```

Comment:
only resistors should use R as the reference designator

```suggestion
          name="K1"
```